src/Illuminate/Http/Request.php

```diff
@@ -4,6 +4,7 @@
 
 use Closure;
 use ArrayAccess;
+use Illuminate\Support\Facades\URL;
```

Comment:
Best not to use Facades in the classes, but inject them in the constructor if you need to.